src/ucs/config/parser.c

```diff
@@ -1429,16 +1429,21 @@ ucs_config_parser_set_value_internal(void *opts, ucs_config_field_t *fields,
             /* Possible override with my prefix */
             if (table_prefix != NULL) {
                 status = ucs_config_parser_set_value_internal(var, sub_fields,
-                                                             name, value,
-                                                             table_prefix, 0);
+                                                              name, value,
+                                                              table_prefix, 0);
                 if (status == UCS_OK) {
                     ++count;
                 } else if (status != UCS_ERR_NO_ELEM) {
                     return status;
                 }
             }
-        } else if (((table_prefix == NULL) || !strncmp(name, table_prefix, prefix_len)) &&
-                   !strcmp(name + prefix_len, field->name))
+        } else if (/* check exact match */
+                   (((table_prefix == NULL) ||
+                     !strncmp(name, table_prefix, prefix_len)) &&
+                    !strcmp(name + prefix_len, field->name)) ||
+                   /* ignore prefix */
```

Comment:
maybe use "fnmatch" for checking?

Reply:
@yosefe It will add variability like fully supported wildcards, I'm not sure if this is what we want in addition to current behavior of prefix+name, up to you.